scripts/rls-breakage-probe.sql

```
@@ -0,0 +1,77 @@
+-- Run:  psql "$DATABASE_URL" -v ON_ERROR_STOP=1 -f scripts/rls-breakage-probe.sql
+--
+-- Each probe prints a BEFORE (no RLS) and an AFTER (FORCE RLS, no GUC set).
+-- A background scheduler / the migration runner / the auth bootstrap all have
+-- NO request context, therefore NO app.clinic_id. AFTER=0 is the breakage.
+--
+-- NOTE: the app role (vettrack) OWNS these tables and is not BYPASSRLS, so
+-- ENABLE alone is a no-op for it. FORCE is the operative flag.
+
+\echo '=== 1. AUTH BOOTSTRAP DEADLOCK (server/middleware/tenant-context.ts:44-51) ==='
+SELECT count(*) AS bootstrap_BEFORE
+  FROM (SELECT clinic_id FROM vt_users WHERE clerk_id='dev-user-alpha' AND deleted_at IS NULL LIMIT 1) x;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)